experiment/generate_tests.py

```diff
@@ -75,7 +75,7 @@ def write_prow_configs_file(output_file, job_defs):
     with open(output_file, 'w') as fp:
         yaml.dump(
             job_defs, fp, Dumper=yaml.RoundTripDumper, width=float("inf"))
-        fp.write('\n')
+        #fp.write('\n')
```

Comment:
What is this?

Reply:
will fixup to actually remove this line.

I don't know why we decided to do this before, but the yamllint  presubmit doesn't allow any blank lines so I needed to remove it :/

Reply:
removed